object_manager.go

```diff
@@ -25,15 +26,15 @@ type IBObjectManager interface {
 type ObjectManager struct {
 	connector IBConnector
 	cmpType   string
-	tenantID  string
+	TenantID  string
```

Comment:
I made TenantID exportable so that I can use the Docker ID in the Lock https://github.com/ishantt/docker-infoblox/blob/swarm/driver/lock.go#L43